frontend/app/routes/_gcweb-app.personal-information.address.edit.tsx

```diff
@@ -0,0 +1,37 @@
+import { Form, Link } from '@remix-run/react';
+
+export default function ChangeAddress() {
+  return (
+    <>
+      <h1 id="wb-cont" property="name">
+        Change Address
+      </h1>
+      <Form method="post">
+        <div className="form-group">
+          <label htmlFor="homeAddress" className="required">
+            Home address <strong className="text-danger mrgn-lft-sm">(required)</strong>
+          </label>
+          <input id="homeAddress" name="homeAddress" className="form-control" />
+        </div>
+
+        <div className="form-group">
+          <label htmlFor="mailingAddress" className="required">
+            Mailing address<strong className="text-danger mrgn-lft-sm">(required)</strong>
```

Comment:
```suggestion
            Mailing address <strong className="text-danger">(required)</strong>
```

Reply:
You removed the class name in your latest commit, but didn't add the space after the word `address`. You need a space otherwise it will render `Mailing address(required)`

Reply:
updated in commit https://github.com/DTS-STN/canadian-dental-care-plan/pull/97/commits/3c10b1aad9850d114010ae776de8b2781edb4916